src/api/helpers/map-currencies/index.ts

```diff
@@ -20,12 +20,24 @@ export const getSupportedCurrencies = (currencies: Array<Currency>) => {
  * 1) Filter supported currencies.
  * 2) Sort the currencies alphabetically.
  * @param {Array} Array of currency objects
+ * @param {Boolean} allCurrencies if all currencies should be returned
```

Comment:
Can we update the documentation to describe the new condition, "only supported currencies" or "all currencies"? it currently describes it will return and sort supported currencies.